paywall/src/unlock.js/CheckoutUIHandler.ts

```diff
@@ -1,6 +1,24 @@
 import IframeHandler from './IframeHandler'
 import { PostMessages } from '../messageTypes'
-import { PaywallConfig } from '../unlockTypes'
+import { PaywallConfig, Balance } from '../unlockTypes'
+
+// Arbitrarily defined number of tokens to assume a managed user
+// account holds for any given stablecoin
+export const defaultBalance = '35'
```

Comment:
I mentioned in discussion with @julien51 that because we are unlikely to approve any especially expensive locks, we can start by just assuming something like 35DAI.

Reply:
Can we make this a constant? This would make re-using easier... just in case!